workers/unity/Packages/com.improbable.gdk.tools/ScriptableGdkToolsConfiguration.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public class ScriptableGdkToolsConfiguration : ScriptableObject
+    {
+        public string SchemaStdLibDir = DefaultValues.SchemaStdLibDir;
+        public List<string> SchemaSourceDirs = new List<string> { DefaultValues.SchemaSourceDir };
+        public string CodegenOutputDir = DefaultValues.CodegenOutputDir;
+
+        private const string AssetPath = "Assets/Config/GdkToolsConfiguration.asset";
+
+        internal List<string> Validate()
+        {
+            var errors = new List<string>();
+            if (string.IsNullOrEmpty(SchemaStdLibDir))
+            {
+                errors.Add($"{GdkToolsConfigurationInspector.SchemaStdLibDirLabel} cannot be empty!");
+            }
+
+            if (string.IsNullOrEmpty(CodegenOutputDir))
+            {
+                errors.Add($"{GdkToolsConfigurationInspector.CodegenOutputDirLabel} cannot be empty!");
+            }
+
+            if (SchemaSourceDirs.Any(string.IsNullOrEmpty))
+            {
+                errors.Add($"Cannot have any empty entry in {GdkToolsConfigurationInspector.SchemaSourceDirsLabel}");
+            }
+
+            if (SchemaSourceDirs.Count == 0)
+            {
+                errors.Add($"Cannot have no entrys in {GdkToolsConfigurationInspector.SchemaSourceDirsLabel}");
```

Comment:
Double negative: "Cannot have no"
*entries

How about "You must have at least one item in..."